### DISCUSSION TITLE
关于组织仓库更新维护的说明

### DISCUSSION THREAD

**Discussion by yetist** [loongarch64/website#35] [Announcements]
@loongarch64/dev-team 

大家好，在本组织中创建的仓库，有相当一部分已经提交到上游，这些仓库已经完成了历史使命，近期将整理这些仓库，对于已经不再需要继续工作的仓库，将进行归档处理，按以往的操作，将会归档到 https://github.com/loongarch64-archive 组织下。

请各位列出目前及未来还需要继续存在的仓库名称，这些仓库将继续保留，以支持开发、review 和 PR 等操作，当完成历史使命之后，也会在未来某个时间点进行归档。

**希望大家在2023年5月5日前完成，谢谢。**

**示例**

需要保留的项目：
- grub: 未提交到上游，还需要继续开发和review
- systemd: 还有一个特性在维护，等待 libseccomp 发布版本后，将向上游发起 PR
- ipxe: review中，部分补丁合并，其余补丁在等gcc13发布之后继续。
- ...

可归档的项目：
- file: 所有补丁已经合并到上游，可归档。
- ...

**Comment by foxsen** [comment]
OS大赛那些题目都可以删除了。
